script.js

```diff
@@ -30,9 +30,10 @@ function showMessage(input, message, type) {
         return false;
     }
     //validate email RegEx 
-    const emailRegex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    const emailRegex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-z\-0-9]+\.)+[a-z]{2,}))$/;
   
-    const email = input.value.toLowerCase().trim();
+    
+    const email = input.value.trim();
   
     if (!emailRegex.test(email)) {
       return showError(input, invalidMsg);
```

Comment:
- [ ] As requested by the previous reviewer, please ensure your form is not sent when using Uppercase letters in the email area.